doc_template/examples_root/examples/nb/ecephys_session.ipynb

```diff
@@ -86,21 +86,7 @@
     "#     manifest=manifest_path, \n",
```

Comment:
<p>Uh... will this be resolved?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/AllenSDK/pull/863/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
that file has not been regenerated since running speed was updated. We are getting new spike sorted data for most sessions, so it makes sense to rerun the pipeline once those are ingested